app/helpers/number_helper.rb

```diff
@@ -69,8 +69,6 @@ def mhz_to_human_size(size, *args)
     nil
   end
 
-  private
-
```

Comment:
Yeah, I'm so on the fence about it.

> false sense of security to people who don't realize these nuances 

Yeah...Part of me thinks "If you don't know about this in Ruby, the code shouldn't be the thing that's teaching you.  You just need to buck up and learn it", but then another part of me is thinking "We need to make sure the code base handles all of the cases and aids the developer".  I dunno.